### ISSUE TITLE
getSessionToken recently stopped resolving

### ISSUE THREAD

**Issue by tobymurray** [Shopify/shopify-app-bridge#185]
## Describe the bug

`app_bridge_utils.getSessionToken(app)` does not resolve

### To Reproduce

I'm not sure what the difference is (application install date, perhaps?), but for some stores this code does not work
```
   <script src="https://unpkg.com/@shopify/app-bridge@3"></script>
   <script src="https://unpkg.com/@shopify/app-bridge-utils@3"></script>

   <script>
      var host = "{{ host }}";
      var AppBridge = window['app-bridge'];
      var app_bridge_utils = window['app-bridge-utils'];
      var app = AppBridge.createApp({
         apiKey: "{{ api_key }}",
         host: host
      });
      promise = app_bridge_utils.getSessionToken(app)
```
`promise` never resolves.

### Expected behaviour

Whether successful or not, the promise should resolve with either a token or an error.

### Packages and versions

   <script src="https://unpkg.com/@shopify/app-bridge@3"></script>
   <script src="https://unpkg.com/@shopify/app-bridge-utils@3"></script>

## Platform

- App: Firefox

### Additional context
This is vanilla JavaScript inline that hasn't changed since July. I see on NPM that app_bridge_utils has been deprecated, but it's not obvious what the equivalent to the new code is as there's no information there?

**Comment by charlesdobson** [comment]
Hi @tobymurray 👋🏼

Can you try importing `getSessionToken` directly from `app-bridge` and let me know if you're seeing anything different?

```
import { getSessionToken } from "@shopify/app-bridge/utilities";
```

We expose all of `@shopify/app-bridge-utils` through the main `@shopify/app-bridge` library now.

**Comment by tobymurray** [comment]
tl;dr: the behavior that exposes this bug is a configuration error, the promise should still resolve as an error

I believe I've found the root cause of why this has changed for installations of my application -  it looks like the general issue is around the new URLs. Manually configuring this with a base64 encoded host that looks like `toby-test-store2.myshopify.com/admin` elicits the above issue. Swapping it to a base64 encoded host that looks like `admin.shopify.com/store/toby-test-store2` "works".

I see the documentation [here](https://shopify.dev/docs/apps/tools/app-bridge/getting-started/app-setup#initialize-shopify-app-bridge-in-your-app) that says:
> The host of the specific shop that's embedding your app. This value is provided by Shopify as a URL query parameter that's appended to your application URL when your app is loaded inside the Shopify admin.

So it looks like historically I was using the `shop` parameter with `/admin` tacked onto it and that has broken as `shop` and the admin path are no longer the same.

**Comment by andriusvo** [comment]
@tobymurray How you handled this issue? Because it seems the same for me. Want to develop locally and specified specific host - not working :(
